custom/icds/rules/custom_criteria.py

```diff
@@ -76,6 +80,26 @@ def ccs_record_case_has_future_edd(case, now):
     return todays_date(now) < edd
 
 
+def ccs_record_case_is_availing_services(case, now):
+    """
+    This filters to ccs record cases where the relevant child person case is both registered and not migrated.
+    """
+    mother = mother_person_case_from_ccs_record_case(case)
+    children = child_person_cases_from_mother_person_case(mother)
+
+    add = case.get_case_property('add')
+    children = [child for child in children if child.get_case_property('dob') == add]
+
+    if not children:
+        return False
+
+    return any([
```

Comment:
The mother person case can also be migrated or not registered. Is it intentional to only account for the child?

Reply:
Additionally there are ccs record cases without children (pregnant women). Are these also intentionally excluded?

Reply:
Confirmed with app team that both are intentional.